### PR TITLE
Release note for #667

### PR DESCRIPTION
**Pull request by aswinsuryan**
Release note for  https://github.com/submariner-io/cloud-prepare/issues/667

### CODE REVIEW

File: release-notes/2023058-ovn-gw-node.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
+Fix issue with spawning gateway nodes in cloud prepare for OVN-Kubernetes CNI in Openstack
```
Comment (sridhargaddam) [inline release-notes/2023058-ovn-gw-node.md]: I'll let @dfarrell07 @tpantelis to comment on this.

Personally, my suggestion would be ....
```
Fix issues while spawning Gateway nodes during cloud prepare for clusters deployed on OpenStack environment running OVN-Kubernetes CNI.
```

File: release-notes/2023058-ovn-gw-node.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
```
Comment (sridhargaddam) [inline release-notes/2023058-ovn-gw-node.md]: Also please rename the file to `20230508-ovn-gw-node.md`

File: release-notes/2023058-ovn-gw-node.md (hunk 1 above)
Comment (aswinsuryan) [reply release-notes/2023058-ovn-gw-node.md]: Done.

Comment (tpantelis) [reply release-notes/2023058-ovn-gw-node.md]: That's fine except I still don't think present tense makes sense, ie "Fixed issues ....".  "Fix issues" sounds like it's still pending or an instruction to do so. But that's my opinion...

File: release-notes/2023058-ovn-gw-node.md (hunk 2 above)
Comment (aswinsuryan) [reply release-notes/2023058-ovn-gw-node.md]: done.

File: release-notes/20230508-ovn-gw-node.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
+Fix issues while spawning Gateway nodes during cloud prepare for clusters deployed on OpenStack environment running OVN-Kubernetes CNI.
```
Comment (tpantelis) [inline release-notes/20230508-ovn-gw-node.md]: I still don't think present tense makes sense, ie "Fixed issues ....". "Fix issues" sounds like it's still pending or an instruction to do so. But that's my opinion...

Comment (sridhargaddam) [reply release-notes/20230508-ovn-gw-node.md]: Exactly, I had the same thought. But when I had a quick look at the existing release notes[1], I see that it uses "Fix ..." 

[1] https://submariner.io/community/releases/

Comment (tpantelis) [reply release-notes/20230508-ovn-gw-node.md]: Some use "Fixed". I think we should change all "Fix" to "Fixed".

Comment (aswinsuryan) [reply release-notes/20230508-ovn-gw-node.md]: Done.

**Review by sridhargaddam — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**